apps/cyberstorm-storybook/LinkLibrary.tsx

```diff
@@ -3,6 +3,7 @@ import React from "react";
 
 interface CyberstormLinkProps {
```

Comment:
Not really related to the task at hand and therefore not a blocker, but I find it somewhat confusing that we have this intermediary `CyberstormLinkProps` that's used only to define the `LinkProps`. My first thought was that some specific meaning is being conveyed with the naming but I can't figure out what it might be. Furthermore this differs from the way the `LinkLibrary` in `cyberstorm-nextjs` defines the interface, for no particular reason. I'd prefer if they'd use the same way just for consistency.